### PR TITLE
Little Update v2.3

### PR DESCRIPTION
**Pull request by ghost**
🇳🇱 Added dutch language.
📑 Updated the instructions.
📝 Fixed some spelling mistakes.

### CODE REVIEW

File: metadata.desktop [hunk 1]
```diff
@@ -17,3 +17,7 @@ X-KDE-PluginInfo-Version=2.2
 X-KDE-PluginInfo-Category=Application Launchers
 X-KDE-PluginInfo-License=GPL-2.0+
 X-KDE-PluginInfo-EnabledByDefault=true
+
+Name[nl]=Beter applicatiemenu
+Comment[nl]=Een verbeterde vork van het Kicker-toepassingsmenu, een opstartprogramma dat is gebaseerd op trapsgewijze pop-upmenu's
```
Comment (HimDek) [inline metadata.desktop:22]: * Please put all the `Name`s together below the English `Name`.
* Please put all the `Comment`s together below the English `Comment`

File: metadata.json [hunk 2]
```diff
@@ -12,11 +12,14 @@
         ],
         "Category": "Application Launchers",
         "Description": "An improved fork of Kicker Application Menu which is a launcher based on cascading popup menus",
+        "Category[nl]": "Beter applicatiemenu",
+        "Description[nl]": "Een verbeterde vork van het Kicker-toepassingsmenu, een opstartprogramma dat is gebaseerd op trapsgewijze pop-upmenu's",
```
Comment (HimDek) [inline metadata.json:16]: * Please put `Category[nl]` below English `Category`
* Please put `Description[nl]` below English `Description`

File: metadata.json [hunk 3]
```diff
@@ -12,15 +12,18 @@
         ],
         "Category": "Application Launchers",
         "Description": "An improved fork of Kicker Application Menu which is a launcher based on cascading popup menus",
+        "Category[nl]": "Beter applicatiemenu",
```
Comment (HimDek) [inline metadata.json]: Category should be a generic name i.e `Application Launchers` and not the specific name i.e `Better Application Launcher`

File: metadata.json [hunk 4]
```diff
@@ -12,15 +12,18 @@
...
+        "Description[nl]": "Een verbeterde vork van het Kicker-toepassingsmenu, een opstartprogramma dat is gebaseerd op trapsgewijze pop-upmenu's",
```
Comment (HimDek) [inline metadata.json:16]: * Please put `Category[nl]` below English `Category`
* Please put `Description[nl]` below English `Description`

File: metadata.desktop [hunk 5]
```diff
@@ -13,7 +13,11 @@ X-Plasma-RequiredExtensions=LaunchApp
...
 X-KDE-PluginInfo-Author=various
 X-KDE-PluginInfo-Name=org.kde.plasma.betterkicker
-X-KDE-PluginInfo-Version=2.2
+X-KDE-PluginInfo-Version=2.3
 X-KDE-PluginInfo-Category=Application Launchers
 X-KDE-PluginInfo-License=GPL-2.0+
 X-KDE-PluginInfo-EnabledByDefault=true
+
+Name[nl]=Beter applicatiemenu
+Comment[nl]=Een verbeterde vork van het Kicker-toepassingsmenu, een opstartprogramma dat is gebaseerd op trapsgewijze pop-upmenu's
```
Comment (HimDek) [inline metadata.desktop]: * Please put all the `Name`s together below the English `Name`.
* Please put all the `Comment`s together below the English `Comment`

File: README.md [hunk 6]
```diff
@@ -56,6 +57,11 @@ The search bar uses `Plasma Search`, which is the same search provider used in K
   <pre>kpackagetool5 -t Plasma/Applet --install</pre>
 </p>
 
+<p align="center">Now make sure you have plasma-framework package installed by execute the following command:</p>
+<p align="center">
+  <pre>sudo apt install plasma-framework</pre>
+</p>
```
Comment (HimDek) [inline README.md]: * Put this part in the `Prerequisites:` section below or within `* Plasma-Framework Package`

**Review by HimDek — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
The changes has been made.

File: metadata.desktop [hunk 7]
```diff
@@ -13,7 +15,7 @@ X-Plasma-RequiredExtensions=LaunchApp
 
 X-KDE-PluginInfo-Author=various
 X-KDE-PluginInfo-Name=org.kde.plasma.betterkicker
-X-KDE-PluginInfo-Version=2.2
+X-KDE-PluginInfo-Version=2.3
```
Comment (HimDek) [inline metadata.desktop]: No need to change the version number. I change it when making a new release.

**Review by HimDek — CHANGES_REQUESTED** [review]
Again not sure if [this](https://github.com/Tim-Notenbomer/Better-Kicker-for-Plasma/blob/master/contents/locale/nl/LC_MESSAGES/plasma_applet_org.kde.plasma.betterkicker.mo) file makes any difference.

**Comment by ghost** [comment]
I think that's everything that needed to be changed.

**Review by HimDek — APPROVED** [review]

**Comment by HimDek** [comment]
Do you know if [this](https://github.com/Tim-Notenbomer/Better-Kicker-for-Plasma/blob/master/contents/locale/nl/LC_MESSAGES/plasma_applet_org.kde.plasma.betterkicker.mo) file makes any difference?

**Comment by ghost** [comment]
I keep forgetting about answering that question but yeah, here is an image to showcase the difference
https://i.imgur.com/IBtKOB6.png

**Pull request closed without merging**